workbench-core/base/src/interfaces/FilterRequest.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { QueryParameterFilter } from './queryParameterFilter';
+
+export interface FilterRequest {
```

Comment:
nit: file name should be `filterRequest.ts`

Reply:
fixed in commit

Reply:
I see the filenames in import statements are fixed, but are the files themselves now starting with lowercase? they still look like uppercase in the Files tab here.

Reply:
^^ I don't think you actually changed the file names which is why the build and test GH action failed btw.